backend/load_real_data.py

```
@@ -0,0 +1,218 @@
+
+# Load environment variables
+load_dotenv()
+
+def get_db_connection():
+    """Get database connection"""
+    db_path = os.path.join(os.path.dirname(__file__), 'instance', 'opportunities.db')
+    conn = sqlite3.connect(db_path)
+    conn.row_factory = sqlite3.Row
+    return conn
+
+def fetch_sam_opportunities(api_key, limit=100):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)